### PR TITLE
ceph.in, mgr: misc cleanups

### PR DESCRIPTION
**Pull request by Liuchang0812**

### CODE REVIEW

File: src/mgr/MgrStandby.cc [hunk 1]
```diff
@@ -147,21 +147,21 @@ int MgrStandby::init()
...
-  dout(1) << state_str() << dendl;
+  dout(4) << "state: " << state_str() << dendl;
 
   set<string> modules;
   PyModules::list_modules(&modules);
   bool available = active_mgr != nullptr && active_mgr->is_initialized();
   auto addr = available ? active_mgr->get_server_addr() : entity_addr_t();
   dout(10) << "sending beacon as gid " << monc.get_global_id()
-	   << " modules " << modules << dendl;
+           << " modules " << modules << dendl;
```
Comment (tchaikov) [inline src/mgr/MgrStandby.cc]: https://github.com/ceph/ceph/blob/master/CodingStyle#L76, and please do not send formatting-only patches unless you happen to touch that area.

File: src/mgr/MgrStandby.cc [hunk 2]
```diff
@@ -147,21 +147,21 @@ int MgrStandby::init()
 void MgrStandby::send_beacon()
 {
   assert(lock.is_locked_by_me());
-  dout(1) << state_str() << dendl;
+  dout(4) << "state: " << state_str() << dendl;
```
Comment (tchaikov) [inline src/mgr/MgrStandby.cc]: please put the rationale in the commit message.

File: src/mgr/PyState.cc [hunk 3]
```diff
@@ -172,6 +172,7 @@ ceph_send_command(PyObject *self, PyObject *args)
         c);
     return nullptr;
   } else {
+    delete c;
```
Comment (tchaikov) [inline src/mgr/PyState.cc]: see #16227

Comment (Liuchang0812) [inline src/mgr/PyState.cc]: cool, I will revert this commit

File: src/ceph.in [hunk 4]
```diff
@@ -886,18 +886,15 @@ def main():
         if not timeout:
             timeout = 5
 
-        hdr('Monitor commands:')
-        print('[Contacting monitor, timeout after %d seconds]' % timeout)
-
-    if childargs and childargs[0] == 'ping':
+    if childargs and childargs[0] == 'ping' and not parsed_args.help:
```
Comment (Liuchang0812) [inline src/ceph.in]: fix "ceph ping --help" problem. We should list commands only with "ping"  prefix. 

```
 Monitor commands:
 =================
"ping" requires a monitor name as argument: "ping mon.<id>"
```

Comment (Liuchang0812) [inline src/ceph.in]: @tchaikov @dmick does it make sense? please take a look. thanks!

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-07-10_16:55:37-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**